### PR TITLE
Add 2022-03-15 minutes

### PR DESCRIPTION
**Pull request by lachie83**
Signed-off-by: Lachlan Evenson <lachlan.evenson@microsoft.com>

### CODE REVIEW

File: minutes/2022-03-15.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   - If people are challenging a yes/no to the rubric - where should we have that discussion (Nisha)
+   - I would like to the proposal to be the one stop shop (Brandon)
+   - I thought of the proposal docs to be a snapshot and not to be continuously updated (Josh)
+   - I don't think bloating the proposal document is the right
+   - DECISION to punt the feedback decision until later and track the comments on the proposal addition to the rubric PR
+- Proposal C discussion
+   -  Introduces a new node manifest
+   -  node manifest looks very similar to index however it adds a description and a manifest blob
+   -  Questions
+   -  When we're looking at objects field, are they manifest or objects or both and how are they accessed and interact? (Brandan)
```
Comment (sudo-bmitch) [inline minutes/2022-03-15.md:42]:
```suggestion
   -  When we're looking at objects field, are they manifest or blobs or both and how are they accessed and interact? (Brandon)
```

File: minutes/2022-03-15.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   -  If object can point to a blob or other objects it gets confusing for the client to understand which API to send the request to
+   -  If you're writing a client that's trying to construct the tree, I don't know how to do it.
+   -  If I push a v2 image manifest, do I need to push a v3 at the same time or is it created and I could. Can I point to v2 manifests from v3 APIs? (Jason)
+   -  Confusion about the addition of the objects? Can you help explain when they are used (Josh)
+   -  Can you walk through the series of pushing to understand the process. What is the entrypoint? (Josh) The reference field is still causing confusing
+   -  Is this similar to what we tried to do in oras artifacts? Ref: https://github.com/oras-project/artifacts-spec/pull/75 (Steve)
+   -  How do you recommend we would filter for objects of a specific type? (Sajay)
+       -  Artifact type has been specifically left out because there were use cases where IANA types were not favored
+   -  What are you thoughts on backwards compatibility? (Brandon)
+   -  If I copy this a registry th at supports v3 to a registry that supports v2? (Brandon)
```
Comment (sudo-bmitch) [inline minutes/2022-03-15.md:52]:
```suggestion
   -  If I copy this from a registry that supports v3 to a registry that supports v2? (Brandon)
```

**Review by sudo-bmitch — CHANGES_REQUESTED** [review]
Just a couple typos. Thank you for doing the note taking this week Lachlan!

File: minutes/2022-03-15.md (hunk 2 above)
Comment (lachie83) [reply minutes/2022-03-15.md:52]: Thanks Brandon - Will fix

**Review by sudo-bmitch — APPROVED** [review]

**Review by SteveLasker — APPROVED** [review]

**Review by imjasonh — APPROVED** [review]

**Pull request merged**